packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.constants.js

```diff
@@ -6,8 +6,12 @@ export const KUBECONFIG_URL =
 export const KUBECTL_URL =
   'https://kubernetes.io/docs/reference/kubectl/overview/';
 
+export const VERSIONS_GUIDE_URL =
+  'https://docs.ovh.com/gb/en/kubernetes/eos-eol-policies/';
```

Comment:
no other translations?

Reply:
Only English link is mentioned in Jira

Reply:
But do you think only English users will use the Manager ? 🙂

Reply:
I got a confirmation, its only available in English as of now.

Reply:
please refer MANAGER-4238

Reply:
Should we allow displaying only English documentation though ? French is our main language right now 😃 

Also, it looks like your code is not ready to handle new documentation as them come (I mean, when we will add French documentation, you will have to make a lot of modifications to handle it)

Reply:
@frenautvh @FredericEspiau 

As @radireddy mentionned, it is the original request.

Our team who manage the documentation repository can only provide this guide in EN.

That could be enhance regarding your feedback but should be defined in a separate ticket.
Feel free to create a new one that refer to this.

Reply:
@FredericEspiau Currently we only have EN doc. If I add code to generate doc URL based on user language (enable to support all languages), all doc URL's apart from EN would lead to 404 errors. As @antleblanc mentioned, it might be best to handle it as a different ticket as and when it comes. Let me know If I am wrong.